panic_handlers/console_panic/src/lib.rs

```diff
@@ -0,0 +1,21 @@
+#![no_std]
+use core::fmt::Write;
+use libtock_console::Console;
+use libtock_low_level_debug::{AlertCode, LowLevelDebug};
+use libtock_platform::{ErrorCode, Syscalls};
+use libtock_runtime::TockSyscalls;
+
+/// This handler requires some 0x400 bytes of stack
+
+#[panic_handler]
+fn panic_handler(info: &core::panic::PanicInfo) -> ! {
+    let mut writer = Console::<TockSyscalls>::writer();
+    if writeln!(writer, "{}", info).is_err() {
+        // Signal a panic using the LowLevelDebug capsule (if available).
+        LowLevelDebug::<TockSyscalls>::print_alert_code(AlertCode::Panic);
```

Comment:
I think we should invoke `LowLevelDebug` before printing via the console, and invoke it unconditionally. As much as I'd like to make `libtock-rs` bulletproof, the reality is I expect that users will occasionally see `panic!` invoked in the presence of undefined behavior, and `LowLevelDebug` is a more reliable way to signal an error in that case.

Reply:
Done.